app/components/main-map.js

```diff
@@ -43,8 +43,16 @@ export default Ember.Component.extend({
       const height = el.height();
       const width = el.width();
 
+      const fullWidth = $('.navigation-area').width();
```

Comment:
@chriswhong is this the only div where we can get correct information about the width of the screen?

Reply:
No, it's just the closest one.  Just need something full width so we can calculate the percentage that the details pane will take up.

Reply:
Got it - so I think window width will do.